docs/community/community-membership.md

```diff
@@ -0,0 +1,174 @@
+description: Community organisation and members responsibilities
+<!--- END of page meta data -->
+
+!!!note
+    This document is in progress
+
+This doc outlines the various responsibilities of contributor roles in
+this project.
+
+| Role | Responsibilities | Defined by |
```

Comment:
suggestion : format the table

```
| Role                | Responsibilities                             | Defined by                  |
|---------------------|----------------------------------------------|-----------------------------|
| Everyone            | none                                         | anybody with a belly button |
| Member              | everyone who contributes - code or otherwise | Orion GitHub org member     |
| Approver            | approve accepting contributions              | write permissions on master |
| Project Manager     | management of the project                    | PegaSys                     |
| Project Sponsor     | contribute developer resources               | PegaSys                     |
| Open Source Circle  | OSS support                                  | PegaSys                     |
| Project Evangelist  | promote the project                          | PegaSys                     |
| Benevolent Dictator | decision tie-breaker                         | PegaSys                     |

```

Reply:
Done